reporter/grpc_test.go

```diff
@@ -19,54 +19,61 @@ package reporter
 
 import (
 	"context"
-	"fmt"
 	"log"
-	"math/rand"
 	"os"
-	"strings"
 	"testing"
 	"time"
 
-	"github.com/golang/mock/gomock"
-
 	"github.com/SkyAPM/go2sky"
-	"github.com/SkyAPM/go2sky/reporter/grpc/common"
-	"github.com/SkyAPM/go2sky/reporter/grpc/register"
-	"github.com/SkyAPM/go2sky/reporter/grpc/register/mock_register"
+	"github.com/SkyAPM/go2sky/propagation"
+	v3 "github.com/SkyAPM/go2sky/reporter/grpc/language-agent"
+)
+
+const (
+	sample                = 1
+	traceID               = "6381ed5e780b11ea9ab88c8590aa458a.1586180089765625000.7116577444803184968"
+	parentSegmentID       = "1586180089765670000.6381ed5e780b11ea9ab88c8590aa458a.515964697678206574"
```

Comment:
Why we need a such long ID? If you have switching to UUID, then we just need it. `.` and 3 parts are not required now.
Long trace/segment id would cause performance issue.

Reply:
Done. I wrote a benchmark for UUID and the old method, and the performance is very close.

```bash
cd internal/idgen && go test -bench=. -benchtime=20s

goos: darwin
goarch: amd64
pkg: github.com/SkyAPM/go2sky/internal/idgen
BenchmarkOldGenerateGlobalID-4   	71331856	       336 ns/op
BenchmarkGenerateGlobalID-4      	76960042	       327 ns/op
PASS
ok  	github.com/SkyAPM/go2sky/internal/idgen	50.215s
```

Reply:
300+ ns/op should be good enough. Thanks for providing this.